website/src/components/SearchPage/DownloadDialog/DownloadForm.tsx

```diff
@@ -87,19 +93,32 @@ export const DownloadForm: FC<DownloadFormProps> = ({
               metadataOption,
               {
                   label: <>Raw nucleotide sequences</>,
-                  subOptions: isMultiSegmented ? (
+                  subOptions: (
                       <div className='px-8'>
-                          <DropdownOptionBlock
-                              name='unalignedNucleotideSequences'
-                              options={referenceGenomesSequenceNames.nucleotideSequences.map((segment) => ({
-                                  label: <>{segment}</>,
-                              }))}
-                              selected={unalignedNucleotideSequence}
-                              onSelect={setUnalignedNucleotideSequence}
-                              disabled={dataType !== 1}
-                          />
+                          {isMultiSegmented ? (
+                              <DropdownOptionBlock
+                                  name='unalignedNucleotideSequences'
+                                  options={referenceGenomesSequenceNames.nucleotideSequences.map((segment) => ({
+                                      label: <>{segment}</>,
+                                  }))}
+                                  selected={unalignedNucleotideSequence}
+                                  onSelect={setUnalignedNucleotideSequence}
+                                  disabled={dataType !== 1}
+                              />
+                          ) : undefined}
+                          {richFastaHeaderFields && (
+                              <RadioOptionBlock
+                                  name='richFastaHeaders'
+                                  title='FASTA header style'
+                                  options={[{ label: <>Accession</> }, { label: <>Display name</> }]}
```

Comment:
We shouldn't hardcode "Display name" but due to limitation of time, I think it's fine to merge as is and address it in a follow-up issue: #3818